<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dungeon Kristóf</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget{
	background-image: url(:/resources/img/brick_background.png);
}</string>
   </property>
   <layout class="QVBoxLayout" name="viewport" stretch="10">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item alignment="Qt::AlignHCenter">
     <widget class="QWidget" name="menuWidget" native="true">
      <layout class="QVBoxLayout" name="menuVLayout">
       <property name="spacing">
        <number>30</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>40</number>
       </property>
       <item>
        <widget class="QLabel" name="titleLabel">
         <property name="font">
          <font>
           <pointsize>48</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel#titleLabel
{
	color: #dfdfdf;
}</string>
         </property>
         <property name="text">
          <string>Dungeon Kristóf</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="newgameBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Cooper Black</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton[flat=true]{
background-color: rgba(89, 22, 9,200);
color :  #dfdfdf;
border: 4px solid #cdcdcd;
border-radius: 20px;
}

QPushButton:pressed[flat=true]{
background-color: rgba(89, 22, 9,200);
color :  #eef205;
border: 4px solid #eef205;
border-radius: 20px;
}</string>
         </property>
         <property name="text">
          <string>New Game</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="charcreateBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Cooper Black</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton[flat=true]{
background-color: rgba(89, 22, 9,200);
color :  #dfdfdf;
border: 4px solid #cdcdcd;
border-radius: 20px;
}

QPushButton:pressed[flat=true]{
background-color: rgba(89, 22, 9,200);
color :  #eef205;
border: 4px solid #eef205;
border-radius: 20px;
}</string>
         </property>
         <property name="text">
          <string>Create Character</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadGameBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Cooper Black</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton[flat=true]{
background-color: rgba(89, 22, 9,200);
color :  #dfdfdf;
border: 4px solid #cdcdcd;
border-radius: 20px;
}

QPushButton:pressed[flat=true]{
background-color: rgba(89, 22, 9,200);
color :  #eef205;
border: 4px solid #eef205;
border-radius: 20px;
}</string>
         </property>
         <property name="text">
          <string>Load Game</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="quitBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Cooper Black</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton[flat=true]{
background-color: rgba(89, 22, 9,200);
color :  #dfdfdf;
border: 4px solid #cdcdcd;
border-radius: 20px;
}

QPushButton:pressed[flat=true]{
background-color: rgba(89, 22, 9,200);
color :  #eef205;
border: 4px solid #eef205;
border-radius: 20px;
}</string>
         </property>
         <property name="text">
          <string>Quit</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
